plugins/javadoc/src/main/kotlin/javadoc/JavadocPageCreator.kt

```diff
@@ -189,5 +176,14 @@ open class JavadocPageCreator(
         }
         return contents
     }
+
+    private fun DParameter.brief(sourceSet: DokkaSourceSet? = highestJvmSourceSet): List<ContentNode> =
+        briefFromContentNodes(paramsToContentNodes(sourceSet).dropWhile { it is ContentDRILink })
+
+    private fun ContentNode.asJavadocNode(): JavadocSignatureContentNode =
+        (this as ContentGroup).children.firstOrNull() as? JavadocSignatureContentNode ?: throw IllegalStateException("JavadocPageCreator should be used only with JavadocSignatureProvider")
```

Comment:
Hint: I would rather go with sth like "No content for javadoc signature found"

Also, please, make `ContentComposite` implement `WithChildren<ContentNode>`, so that you can use 
`(this as ContentGroup).firstChildOfTypeOrNull< JavadocSignatureContentNode >()` here

Reply:
done x2